src/components/Mapbox.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useRef, useEffect } from 'react';
+import mapboxgl from 'mapbox-gl';
+import { Map, Marker, MapboxOptions, MarkerOptions, LngLatBounds } from 'mapbox-gl';
```

Comment:
why do we have two imports for mapbox-gl?

Reply:
I think I need to keep the general import for the accessToken on ln. 62-- is there another way to do it?  (if I keep it, ill move it down so they're on the same line

Reply:
ahh yeah makes sense! we can keep it, let's just combine it like `import mapboxgl, { blah } from 'mapbox-gl'`